HousingManagementSystemApi/Startup.cs

```diff
@@ -8,15 +8,20 @@
 
 namespace HousingManagementSystemApi
 {
+    using System.Collections.Generic;
     using System.Net.Http;
     using Gateways;
+    using Hackney.Shared.Asset.Domain;
     using HousingRepairsOnline.Authentication.DependencyInjection;
     using UseCases;
 
     public class Startup
     {
         private const string HousingManagementSystemApiIssuerId = "Housing Management System Api";
-
+        public static IEnumerable<AssetType> EligibleAssetTypes = new[]
```

Comment:
if i make this configurable i think this will have to be a string and not IEnumerable

Reply:
I believe you mean it would be a `string` and not `AssetType`.

**TL;DR It's fine the way it is.**

Configuration values can be the values of Enums.
If a single Enum value is value is required, then the value should be a string/text of the value.
However, if multiple Enums values are required to be specified in a single configuration value, the following would be required:
- The [FlagsAttribute](https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=net-6.0) applied to the Enum
- The configuration value specified using comma separated values of the Enum values.
- [Optional, but recommended] Enum values be assigned unique, non-combinable integer values to avoid accidental combinations, i.e. 0, 1, 2, 4, 8 (easier to use [bitwise shifting](https://www.alanzucconi.com/2015/07/26/enum-flags-and-bitwise-operators/#:~:text=Ice%3B%20%20//%20Toggle%20ice-,Bitwise%20shifts,-The%20last%20two) to specify these values)

So addition info on this [StackOverflow answer](https://stackoverflow.com/a/13176549).

As this enum, `AssetType`, comes from an external library which making changes to would be a larger scope of work, this is fine the way it is.